test/sparse/sparse.jl

```diff
@@ -1807,7 +1807,7 @@ end
     @test String(take!(io)) ==  string("6×3 SparseMatrixCSC{Float64,$Int} with 18 stored entries:\n  [1, 1]",
                                        "  =  1.0\n  [2, 1]  =  1.0\n  ⋮\n  [5, 3]  =  1.0\n  [6, 3]  =  1.0")
 
-    ioc = IOContext(io, :displaysize => (9, 80))
+    ioc = IOContext(io, displaysize = (9, 80))
```

Comment:
OK thanks; I don't believe it would be noticeable in test code, but in 893a65bddea0bd89c3df2eb0cf8727b1f967738d Jameson did exactly the opposite for other test files, apparently in a performance related PR -- but he may have done it in test files only for stylistic reasons too ;-)